WordPress/src/main/java/org/wordpress/android/ui/mediapicker/MediaPickerActivity.kt

```diff
@@ -231,6 +237,22 @@ class MediaPickerActivity : LocaleAwareActivity(), MediaPickerListener {
         }
     }
 
+    private fun launchPictureLibrary(multiSelect: Boolean) {
+        WPMediaUtils.launchPictureLibrary(this, multiSelect)
+    }
+
+    private fun launchVideoLibrary(multiSelect: Boolean) {
+        WPMediaUtils.launchVideoLibrary(this, multiSelect)
+    }
+
+    private fun launchMediaLibrary(multiSelect: Boolean) {
+        WPMediaUtils.launchMediaLibrary(this, multiSelect)
+    }
+
+    private fun launchFileLibrary(multiSelect: Boolean) {
+        WPMediaUtils.launchFileLibrary(this, multiSelect)
```

Comment:
we can create only one method in `WPMediaUtils` and pass the set of mime types as a parameter.

Reply:
👍...see [this comment above ](https://github.com/wordpress-mobile/WordPress-Android/pull/12884#discussion_r486983389)